### PR TITLE
remove versions from git dependencies and update bitflags

### PR DESCRIPTION
**Pull request by vhdirk**
For this to work, these modifications are needed on other gtk-rs projects too (e.g. gtk). See https://github.com/gtk-rs/pangocairo/issues/19

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -30,36 +30,14 @@ git = "https://github.com/gtk-rs/lgpl-docs"
...
+bitflags = "1.0"
 
-[dependencies.pango-sys]
-version = "0.6.0"
-git = "https://github.com/gtk-rs/sys"
+glib-sys = { git = "https://github.com/gtk-rs/sys" }
+gobject-sys = { git = "https://github.com/gtk-rs/sys" }
+pango-sys = { git = "https://github.com/gtk-rs/sys" }
+cairo-sys-rs = { git = "https://github.com/gtk-rs/cairo" }
+pangocairo-sys = { git = "https://github.com/gtk-rs/sys" }
```
Comment (GuillaumeGomez) [inline Cargo.toml]: Please remove this empty line.

**Comment by vhdirk** [comment]
Rebased on master

**Comment by EPashkin** [comment]
@vhdirk We have crate branch for this

**Comment by GuillaumeGomez** [comment]
I'm still not sure what to do about this... Any opinion @sdroege?

**Comment by vhdirk** [comment]
The crate branch serves a different purpose. I'm building upon https://github.com/sdroege/gobject-subclass, which specifies it's dependencies differently.
Since you're using git anyway, is there any specific reason why you'd want to specify the exact version as well?

**Comment by GuillaumeGomez** [comment]
I have to admit I never thought about it. Can't recall why we did it like this.

**Comment by vhdirk** [comment]
If there really is a reason for it, I would be totally fine with applying that in https://github.com/sdroege/gobject-subclass (end derivatives). Right now, the only reason I can think of would be to immediately pin down the major revision number for any next release. But that's just guessing.

**Comment by sdroege** [comment]
Maybe just send a PR and make it consistent with all the other gtk-rs repositories (i.e. remove the versions) :)

**Comment by vhdirk** [comment]
I don't mind at all. Just specify what you want it to look like and I'll create PRs to every project that it's needed for.

**Comment by GuillaumeGomez** [comment]
Remove the dependencies version in the gstreamer repositories? I suppose the problem is coming from there no?

**Comment by vhdirk** [comment]
@GuillaumeGomez I'm not using the gstreamer repositories here at all.

**Comment by GuillaumeGomez** [comment]
Euh yes, I meant gobject-subclass... Let's just merge then, I'll fix the release script later on. Thanks!

**Pull request merged**